test/internal/cmd/generate-gateway-api-urls/main.go

```diff
@@ -15,9 +15,10 @@ import (
 	"text/template"
 )
 
-//go:generate go run --tags generate_gateway_api_urls . -crds-standard-url $CRDS_STANDARD_URL -crds-experimental-url $CRDS_EXPERIMENTAL_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
+//go:generate go run --tags generate_gateway_api_urls . -gateway-api-version $GATEWAY_API_VERSION -crds-standard-url $CRDS_STANDARD_URL -crds-experimental-url $CRDS_EXPERIMENTAL_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
```

Comment:
Just a minor nit, but maybe we could rename the command to `generate-gateway-api-consts` then as it's no longer only URLs oriented?

Reply:
Done, PTAL